service_mesh/v2x/ossm-reference-smcp.adoc

```diff
@@ -3,9 +3,29 @@
 include::modules/ossm-document-attributes.adoc[]
 :context: ossm-reference
 
+You can customize your {ProductName} by modifying the default `ServiceMeshControlPlane` (SMCP) resource or by creating a completely custom SMCP resource.  This reference section documents the configuration options available in the SMCP.
```

Comment:
```suggestion
You can customize your {ProductName} by modifying the default `ServiceMeshControlPlane` (SMCP) resource or by creating a completely custom SMCP resource.  This reference section documents the configuration options available for the SMCP resource.
```

Some of my suggestions above use object rather than resource but either is fine